src/v3/variables.ts

```diff
@@ -89,6 +89,14 @@ export function variables(): CompanionVariableDefinition[] {
 			name: 'Timer Blinking',
 			variableId: variableId.TimerBlink,
 		},
+		{
+			name: 'External Message',
+			variableId: variableId.ExternalMessage,
+		},
+		{
+			name: 'Timer Message Secondary Source',
```

Comment:
Ouch, we really made the names here confusing... my bad
How about 
```suggestion
			name: 'Timer Secondary Message',
```